mypaint-brush.c

```diff
@@ -22,6 +22,18 @@
 #include <math.h>
 #include <assert.h>
 
+#define GETTEXT_PACKAGE "libmypaint"
+
+#ifdef HAVE_GETTEXT
```

Comment:
@Jehan @achadwick then we should commit for the moment without glib? and maybe in future adapt this part for enable/disable if the build is supporting glib?